pages/overview/sponsors/index.page.tsx

```diff
@@ -21,7 +21,7 @@ export async function getStaticProps() {
 
 export default function ContentExample({ blocks }: { blocks: any[] }) {
   const newTitle = 'Sponsors';
-
+  const fileRenderType = '_indexmd';
```

Comment:
In case of `Code of Conduct` submodule, the changes point to `tsx` but here for `sponsors` submodule we are pointing to the `_indexmd`, this behavior is not consistent.

Reply:
@DarhkVoyd Resolved the issue